src/Robo/Commands/Tests/BehatCommand.php

```diff
@@ -162,6 +174,72 @@ protected function launchWebDriver() {
     elseif ($this->getConfigValue('behat.web-driver') == 'chrome') {
       $this->launchChrome();
     }
+    elseif ($this->getConfigValue('behat.web-driver') == 'chromium') {
+      $this->launchChromium();
+    }
+  }
+
+  /**
+   * Launches a headless chromium process.
+   */
+  protected function launchChromium() {
+    $this->killChromium();
+    $chromium_bin = $this->findChromium();
+    $this->checkChromiumVersion($chromium_bin);
+    $chromium_host = 'http://localhost';
+    $this->logger->info("Launching headless chromium...");
+    $this->getContainer()
+      ->get('executor')
+      ->execute("'$chromium_bin' --headless --disable-web-security --remote-debugging-port={$this->chromiumPort} {$this->chromiumArgs} $chromium_host")
+      ->background(TRUE)
+      ->printOutput(TRUE)
+      ->printMetadata(TRUE)
+      ->run();
+    $this->getContainer()->get('executor')->waitForUrlAvailable("$chromium_host:{$this->chromiumPort}");
+  }
+
+  /**
+   * Kills headless chromium process running on $this->chromiumPort.
+   */
+  protected function killChromium() {
+    $this->logger->info("Killing running chromium processes...");
+    $this->getContainer()->get('executor')->killProcessByPort($this->chromiumPort);
+  }
+
+  /**
+   * Finds the local Chromium binary.
+   *
+   * @return null|string
+   *   NULL if Chromium could not be found.
+   *
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   *   Throws exception if google-chrome cannot be found.
+   */
+  protected function findChromium() {
+    if ($this->getInspector()->commandExists('chromium')) {
+      return 'chromium';
+    }
+    throw new BltException("Could not find Chromium. Please add an alias for \"chromium\" to your CLI environment.");
+  }
+
+  /**
+   * Verifies that Chromium meets minimum version requirement.
+   *
+   * @param string $bin
+   *   Absolute file path to the chromium bin.
+   *
+   * @throws \Acquia\Blt\Robo\Exceptions\BltException
+   *   Throws exception if minimum version is not met.
+   */
+  protected function checkChromiumVersion($bin) {
+    $version = (int) $this->getContainer()->get('executor')
+      ->execute("'$bin' --version | cut -f3 -d' ' | cut -f1 -d'.'")
```

Comment:
I don't think this one will work.

Just tried locally:

```
$ chromium --version | awk '{print $2}' | cut -f1 -d '.'
64
```

Alternatively, to follow your approach:

```
$ chromium --version | cut -f2 -d ' ' | cut -f1 -d '.'
64
```